stdlib/ballerina-http/src/main/ballerina/http.swagger/annotation.bal

```diff
@@ -10,59 +8,59 @@ import ballerina/http;
 @Field {value: "license: License information for the exposed API."}
 @Field {value: "externalDoc: Additional external documentation."}
 @Field {value: "tags: A list of tags used by the specification with additional metadata"}
-public struct ServiceInformation {
-    string title;
-    string serviceVersion;
-    string description;
-    string termsOfService;
-    Contact contact;
-    License license;
-    DocumentationInformation externalDocs;
-    Tag[] tags;
+public type ServiceInformation {
+    string title,
+    string serviceVersion,
+    string description,
+    string termsOfService,
+    Contact contact,
+    License license,
+    DocumentationInformation externalDocs,
+    Tag[] tags,
     SecurityRequirement[] security;
```

Comment:
Shall we stick to one convention? lets use comma in all places.
Applicable to other type defs as well.

Reply:
Shouldn't we use  comma to separate fields and semi colon to end the fields list? having a comma at the end looks weird.